client/state/signup/steps/website-content/selectors.ts

```diff
@@ -1,10 +1,11 @@
 import 'calypso/state/signup/init';
-import { initialState, WebsiteContentCollection, MEDIA_UPLOAD_STATES } from './schema';
+import { initialState, MEDIA_UPLOAD_STATES } from './constants';
+import type { WebsiteContentCollectionState } from './types';
 
 export interface WebsiteContentStateModel {
 	signup: {
 		steps: {
-			websiteContentCollection: WebsiteContentCollection;
+			websiteContentCollection: WebsiteContentCollectionState;
```

Comment:
Renamed the `WebsiteContentCollection` type to `WebsiteContentCollectionState` to better reflect its meaning.